test/rules/named-spy.js

```diff
@@ -19,6 +19,7 @@ eslintTester.run('named-spy', rule, {
     // Correct use of named spies
     'var onSuccess = jasmine.createSpy("onSuccess")',
     'onSuccess = jasmine.createSpy("onSuccess")',
+    'onSuccess(jasmine.createSpy("callback"))',
```

Comment:
Could you add another spec specifically for this case instead of appending to an existing one?